tvx/lotus/driver.go

```diff
@@ -51,15 +51,12 @@ func (d *Driver) ExecuteMessage(msg *types.Message, preroot cid.Cid, bs blocksto
 		Syscalls:       mkFakedSigSyscalls(vm.Syscalls(ffiwrapper.ProofVerifier)),
 		CircSupplyCalc: nil,
 		BaseFee:        BaseFee,
+		NtwkVersion: func(context.Context, abi.ChainEpoch) network.Version {
+			// TODO: Network upgrade.
+			return network.Version0
+		},
 	}
-	lvm, err := vm.NewVM(vmOpts)
-	if err != nil {
-		return nil, cid.Undef, err
-	}
-	// need to modify the VM invoker to add the puppet actor
-	chainValInvoker := vm.NewInvoker()
-	chainValInvoker.Register(puppet.PuppetActorCodeID, puppet.Actor{}, puppet.State{})
```

Comment:
This has been removed downstream. Did we need it?

Reply:
We need only the ChaosActor (which replaced the PuppetActor) AFAIK, and this is done at https://github.com/filecoin-project/lotus/blob/master/conformance/driver.go#L144

So I think this is fine, and we should actually remove `tvx/lotus/driver.go` as this has now been moved to Lotus - something we can do after this PR.

Reply:
Yes. I'm going to merge #246 which makes tvx use the lotus driver, and then we can erase the driver. I'll push to both PRs directly.